nss-ice-phantoms/WorkforceManagement
# Rr employees

# Description

Complete Issue #3.  Employee details now contains employees assigned computer and training program information.

## Type of change

# Testing Instructions

1. git fetch --all
2. git checkout rr-employees
3. Run BangazonWorkforce
4. Verify Details link by any employee in the list takes you to the details page and displays First Name, Last Name, Full Name, Department Name, Computer and a list of Training Programs.

# Checklist:
